SeaPublicWebsite/Views/EnergyEfficiency/AnswerSummary.cshtml

```diff
@@ -121,7 +121,9 @@
             Value = new()
             {
                 Classes = "govuk-!-width-one-half",
-                Text = GovUkRadioCheckboxLabelTextAttribute.GetLabelText(Model.PropertyData.FloorConstruction)
+                Text = Model.PropertyData.FloorConstruction is FloorConstruction.Mix
```

Comment:
Can we make this into an extension method along the lines of `FloorConstruction.DescriptionForAnswerSummary()` as we want to to go that way anyway in the end?